apps/condo/domains/banking/utils/testSchema/bankAccountGenerate.js

```diff
@@ -0,0 +1,74 @@
+const faker = require('faker')
+
```

Comment:
Are you sure that name of this module `bankAccountGenerate` does not deviates from naming convention of the project? Mayby just `BankAccount.js`?

Reply:
+1